modules/saml/lib/SP/LogoutStore.php

```diff
@@ -14,12 +14,19 @@ class sspmod_saml_SP_LogoutStore {
 	 */
 	private static function createLogoutTable(\SimpleSAML\Store\SQL $store) {
 
-		if ($store->getTableVersion('saml_LogoutStore') === 1) {
+		if ($store->getTableVersion('saml_LogoutStore') === 2) {
+			return;
+		}
+		elseif ($store->getTableVersion('saml_LogoutStore') === 1) {
```

Comment:
I'm being picky here, but if we are modifying the code slightly here, I'd also like to take the chance to follow our coding standard completely. Do you mind moving the `elseif` up one line?

```php
} elseif ($tableVer === 1) {
```

Thanks!